### PR TITLE
refactor(elb/loadbalancer): fix elb loadbalancer lint issues

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  fix elb loadbalancer lint issues
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  fix elb loadbalancer lint issues
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_loadbalancer_test.go'  
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_loadbalancer_test.go -v  -timeout 360m -parallel 4 
=== RUN   TestAccElbV3LoadBalancer_basic 
=== PAUSE TestAccElbV3LoadBalancer_basic 
=== RUN   TestAccElbV3LoadBalancer_withEpsId
=== PAUSE TestAccElbV3LoadBalancer_withEpsId
=== RUN   TestAccElbV3LoadBalancer_withEIP
=== PAUSE TestAccElbV3LoadBalancer_withEIP
=== RUN   TestAccElbV3LoadBalancer_prePaid
=== PAUSE TestAccElbV3LoadBalancer_prePaid
=== CONT  TestAccElbV3LoadBalancer_basic
=== CONT  TestAccElbV3LoadBalancer_withEIP
=== CONT  TestAccElbV3LoadBalancer_withEpsId
=== CONT  TestAccElbV3LoadBalancer_prePaid
    acceptance.go:281: This environment does not support prepaid tests
=== CONT  TestAccElbV3LoadBalancer_withEpsId
    acceptance.go:182: The environment variables does not support Enterprise Project ID for acc tests
--- SKIP: TestAccElbV3LoadBalancer_prePaid (0.00s)
--- SKIP: TestAccElbV3LoadBalancer_withEpsId (0.00s)
--- PASS: TestAccElbV3LoadBalancer_withEIP (33.72s) 
--- PASS: TestAccElbV3LoadBalancer_basic (59.42s) 
PASS
ok      command-line-arguments  59.465s
```

### CODE REVIEW

File: huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go [hunk 1]
```diff
@@ -402,178 +402,192 @@ func resourceLoadBalancerV3Read(_ context.Context, d *schema.ResourceData, meta
...
+	elbClient, err := cfg.ElbV3Client(cfg.GetRegion(d))
 	if err != nil {
-		return diag.Errorf("error creating elb v3 client: %s", err)
+		return diag.Errorf("error creating ELB client: %s", err)
 	}
 
-	//lintignore:R019
+	// lintignore:R019
 	if d.HasChanges("name", "description", "cross_vpc_backend", "ipv4_subnet_id", "ipv6_network_id",
 		"ipv6_bandwidth_id", "ipv4_address", "l4_flavor_id", "l7_flavor_id", "autoscaling_enabled", "min_l7_flavor_id") {
```
Comment (deer-hang) [inline huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go]: maybe define a new variable will be much more clear.
```
	updateLoadBalancerChanges := []string{
		"name", "description", "cross_vpc_backend", "ipv4_subnet_id", "ipv6_network_id",
		"ipv6_bandwidth_id", "ipv4_address", "l4_flavor_id", "l7_flavor_id", "autoscaling_enabled", "min_l7_flavor_id"
	}
```

File: huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go [hunk 2]
```diff
@@ -402,178 +402,192 @@ func resourceLoadBalancerV3Read(_ context.Context, d *schema.ResourceData, meta
...
+			return diag.Errorf("error updating the auto-renew of the loadbalancer (%s): %s", d.Id(), err)
 		}
-		if d.HasChange("l7_flavor_id") {
-			updateOpts.L7Flavor = d.Get("l7_flavor_id").(string)
+	}
+	// update tags
+	if d.HasChange("tags") {
+		elbV2Client, err := cfg.ElbV2Client(cfg.GetRegion(d))
+		if err != nil {
+			return diag.Errorf("error creating elb 2.0 client: %s", err)
```
Comment (deer-hang) [inline huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go]: replace with ``` return diag.Errorf("error creating elb V2 client: %s", err) ```

File: huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go [hunk 3]
```diff
@@ -402,178 +402,192 @@ func resourceLoadBalancerV3Read(_ context.Context, d *schema.ResourceData, meta
...
-			err = common.WaitOrderComplete(ctx, bssClient, resp.OrderID, d.Timeout(schema.TimeoutUpdate))
-			if err != nil {
-				return diag.Errorf("the order is not completed while updating ELB loadbalancer (%s): %#v", resp.LoadBalancerID, err)
-			}
-		} else {
-			_, err = loadbalancers.Update(elbClient, d.Id(), updateOpts).Extract()
-			if err != nil {
-				return diag.Errorf("error updating elb loadbalancer: %s", err)
-			}
+func waitForLoadBalancer(ctx context.Context, d *schema.ResourceData, cfg *config.Config, updateOpts loadbalancers.UpdateOpts,
```
Comment (deer-hang) [inline huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go]: this function need to be redesigned: 
```
func updateLoadBalancer(){}
func waitForLoadBalancerCompleted(){}
```

Comment (houpeng80) [reply huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go]: done

File: huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go (hunk 2 above)
Comment (houpeng80) [reply huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go]: done

File: huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go (hunk 1 above)
Comment (houpeng80) [reply huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go]: it's a good idea, done

**Pull request merged**